libmei/dist/atts_cmn.cpp

```diff
@@ -683,50 +683,50 @@ AttHarpPedalLog::AttHarpPedalLog() : Att()
 
 void AttHarpPedalLog::ResetHarpPedalLog()
 {
-    m_c = harpPedalLog_C_NONE;
-    m_d = harpPedalLog_D_NONE;
-    m_e = harpPedalLog_E_NONE;
-    m_f = harpPedalLog_F_NONE;
-    m_g = harpPedalLog_G_NONE;
-    m_a = harpPedalLog_A_NONE;
-    m_b = harpPedalLog_B_NONE;
+    m_c = HARPPEDALPOSITION_NONE;
```

Comment:
This change looks strange?

Reply:
This is fine. It changed because it because a global data type.

Reply:
OK